app/client/src/components/Post.jsx

```diff
@@ -115,7 +117,7 @@ const Post = (props) => {
         square
       >
         <CardHeader
-          sx={{ backgroundColor: "banner.main", color: "white", py: 1 }}
+          sx={{ backgroundColor: "primary.main", color: "white", py: 1 }}
```

Comment:
Why are we using 'primary' here? 'Banner' is a different colour from 'primary.'

Reply:
I was changing colors around and I forgot to set these back. I'll change that